packages/ui/src/components/molecules/Breadcrumb/styles.scss

```diff
@@ -141,4 +139,28 @@
       @include focus-ring;
     }
   }
+
+  &[data-fs-breadcrumb-is-desktop="true"] {
```

Comment:
👀  do we need it?

Reply:
Yes, it was a behavior implemented to show the correct breadcrumb without losing performance